### ISSUE TITLE
Allow custom properties to export

### ISSUE THREAD

**Issue by krsoninikhil** [victor-rocha/django-csv-exports#7]
This is the simplest package I found for exporting data from admin. 
Problem that I faced was that datetime fields get exported as UTC timezone, even if timezone is enabled in django settings. I tried adding a custom property in model but the way fields are taken in this package [0] only takes database fields.
```python
queryset = queryset.values_list(*field_names)
```
Would you be open to have a PR on adding custom properties in exported data?

[0]: https://github.com/rochapps/django-csv-exports/blob/efcdde401d66f38a64b37afa909bfc16a6c21e9e/django_csv_exports/admin.py#L40

**Comment by vrocha** [comment]
Yes, either that or making the datatime fields export using the enabled timezone setting (my preference).
